features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_SOFTDEVICE/TARGET_NRF52/source/nRF5xPalSecurityManager.cpp

```diff
@@ -681,8 +676,8 @@ ble_error_t nRF5xSecurityManager::secure_connections_oob_request_reply(
 
     uint32_t err = sd_ble_gap_lesc_oob_data_set(
         connection,
-        pairing_cb->own_oob ? &oob_own : NULL,
-        pairing_cb->peer_oob ? &oob_peer : NULL
+        &oob_own,
```

Comment:
Content in `oob_own` or `oob_peer` should be all zeros if not available (according to the BT spec); however the nordic documentation state that we should pass `NULL` when this is the case. I wonder if we should look at the content then pass `NULL` if N/A instead of unconditionally passing the value.

Reply:
ok, i'll take a closer look